src/components/DeleteMotor/DeleteMotor.jsx

```diff
@@ -1,5 +1,81 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { Pagination, Navigation } from 'swiper/modules';
+import { Link } from 'react-router-dom';
+import { motorsItem, deleteMotorItem } from '../../redux/slice/motorSlice'; // Import the deleteMotorItem action
+import NavigationPanel from '../navigation/NavigationPanel';
+import MobileNavbar from '../navigation/MobileNavbar';
+import './DeleteMotor.scss';
+// Import Swiper styles
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
 
-const DeleteMotor = () => <div>DeleteMotor</div>;
+const DeleteMotor = () => {
+  const dispatch = useDispatch();
+  const { motorData } = useSelector((state) => state.motor);
+
+  useEffect(() => {
+    dispatch(motorsItem());
+  }, [dispatch]);
+
+  const handleDelete = (motorId) => {
+    dispatch(deleteMotorItem(motorId));
+  };
+
+  return (
+    <div className="main-container">
+      <MobileNavbar className="mobile-navbar" />
+      <NavigationPanel className="left-panel" />
+      <div className="motor-container">
+        <div className="models-titles">
+          <h1>Latest models</h1>
+          <h5>Please select a model</h5>
+        </div>
+        <div className="swiper-cont">
+          <Swiper
+            navigation
+            modules={[Navigation, Pagination]}
+            className="mySwiper"
+            loopFillGroupWithBlank
+            slidesPerView={3}
+            spaceBetween={20}
+            slidesPerGroup={1}
+          >
+            {motorData.map((item) => (
+              <SwiperSlide key={item.id}>
+                <img
+                  src={item.photo}
+                  alt={item.brand_name}
```

Comment:
As per project requirements, When the user clicks the "Delete item" link in the navigation panel they can see a list of all items with title and "Delete" button. So please kindly make changes to list all motors.

Reply:
done